ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
It is not hard to imagine how many playbooks there is going to be if we take this path. If we break the etcd roles into multiple roles we get overwhelmed. At the same time it will be more difficult to maintain a subset of etcd like playbooks than once playbook with tagged tasks. Instead of creating ``etcd-start``, ``etcd-configure`` and other playbooks included in ``etcd-setup`` one, I would suggest to shift the responsibility of what task gets run into scripts which set appropriate tags.

E.g. 
```sh
$ cat scripts/deploy-etcd.sh
ansible-playbook -i ${INVENTORY} playbooks/etcd-setup.yml --tags all
```

```sh
$ cat scripts/update-etcd.sh
ansible-playbook -i ${INVENTORY} playbooks/etcd-setup.yml --tags "install,restart"
```

```sh
$ cat scripts/restart-etcd.sh
ansible-playbook -i ${INVENTORY} playbooks/etcd-setup.yml --tags "restart"
```

On the level of the entire cluster:

```sh
$ cat scripts/deploy-cluster.sh
ansible-playbook -i ${INVENTORY} playbooks/deploy-cluster.yml
```

```sh
$ cat scripts/update-cluster.sh
ansible-playbook -i ${INVENTORY} playbooks/deploy-cluster.yml --tags="install,restart"
```